app/services/trainees/create_from_dttp.rb

```diff
@@ -527,5 +529,11 @@ def calculate_funding!
     def funding_manager
       @funding_manager ||= FundingManager.new(trainee)
     end
+
+    def update_dttp_sha!
+      trainee.dttp_update_sha = trainee.sha
+
+      trainee.save!(touch: false)
```

Comment:
There are a couple of other places in this file that would set the `updated_at` value to the current datetime, overriding the one you're trying to set from the response.

For example, if the trainee has funding (see `calculate_funding!`) or 'multiple disabilities' (see `add_multiple_disability_text!`).

Also how about if the trainee has degrees? See `Degrees::CreateFromDttp`.

Reply:
That makes a lot of sense! I think wrapping in an  ActiveRecord::Base.no_touching block should solve these issues right?